actions/convert_seconds_to_days_MOD.js

```diff
@@ -143,34 +153,68 @@ module.exports = {
+              m = Math.floor(time % 3600 / 60);
+              s = Math.floor(time % 60);
+                result = ((mo > 0) ? mo + "M " : "") + ((d > 0) ? d + "d " : "") + ((h > 0) ? h + "h " : "") + ((m > 0) ? m + "m " : "") + s + "s";
+            break;
+          case 2: // DHMS
+              d = Math.floor(time / 86400);
+              h = Math.floor(time % 86400 / 3600);
+              m = Math.floor(time % 3600 / 60);
+              s = Math.floor(time % 60);
+                result = ((d > 0) ? d + "d " : "") + ((h > 0) ? h + "h " : "") + ((m > 0) ? m + "m " : "") + s + "s";
+            break;
+          case 3: // HMS
+              h = Math.floor(time / 3600);
+              m = Math.floor(time % 3600 / 60);
+              s = time % 60;
+                result = ((h > 0) ? h + "h " : "") + ((m > 0) ? m + "m " : "") + s + "s";
+            break;
+          case 4: // MS
+              m = Math.floor(time / 60);
+              s = time % 60;
+                result = ((m > 0) ? m + "m " : "") + s + "s";
+            break;
+          case 5: // Formatted MDHMS
+              mo = Math.floor(time / 2592000)
+              d = Math.floor(time % 2592000 / 86400);
+              h = Math.floor(time % 86400 / 3600);
+              m = Math.floor(time % 3600 / 60);
+              s = Math.floor(time % 60);
+                result = ((mo > 0) ? ((mo !== 1 ) ? mo + " Months " : mo + " Month ") : "") + ((d > 0) ? ((d !== 1 ) ? d + " Days " : d + " Day ") : "") + ((h > 0) ? ((h !== 1 ) ? h + " Hours " : h + " Hour ") : "") + ((m > 0) ? ((m !== 1 ) ? m + " Minutes " : m + " Minute ") : "") + ((s !== 1 ) ? s + " Seconds" : s + " Second");
+            break;
+          case 6: // Formatted DHMS
+              d = Math.floor(time / 86400);
+              h = Math.floor(time % 86400 / 3600);
+              m = Math.floor(time % 3600 / 60);
+              s = Math.floor(time % 60);
+                result = ((d > 0) ? ((d !== 1 ) ? d + " Days " : d + " Day ") : "") + ((h > 0) ? ((h !== 1 ) ? h + " Hours " : h + " Hour ") : "") + ((m > 0) ? ((m !== 1 ) ? m + " Minutes " : m + " Minute ") : "") + ((s !== 1 ) ? s + " Seconds" : s + " Second");
+            break;
+          case 7: // Formatted HMS
+              h = Math.floor(time / 3600);
+              m = Math.floor(time % 3600 / 60);
+              s = time % 60;
+                result = ((h > 0) ? ((h !== 1 ) ? h + " Hours " : h + " Hour ") : "") + ((m > 0) ? ((m !== 1 ) ? m + " Minutes " : m + " Minute ") : "") + ((s !== 1 ) ? s + " Seconds" : s + " Second");
+            break;
+          case 8: // Formatted MS
+              m = Math.floor(time / 60);
+              s = time % 60;
+                result = ((m > 0) ? ((m !== 1 ) ? m + " Minutes " : m + " Minute ") : "") + ((s !== 1 ) ? s + " Seconds" : s + " Second");
+            break;
+          default:
+            print("Something went wrong... Check your Convert Seconds to D/H/M/S mod. (#" + cache.index + ")")
```

Comment:
undefined variable

Reply:
Where exactly?

Reply:
`print`

Reply:
`cache.index` should be defined, and anyways, nobody's actually gonna get this unless they break something.

Reply:
"print" is undefined, not talking about cache sir